1_fetch/src/get_nwis_data.R

```diff
@@ -1,19 +1,37 @@
+download_nwis_data <- function(site_nums = c("01427207", "01432160", "01436690", "01466500"), Save_as_csv = FALSE){
```

Comment:
Consider moving these `site_nums` into the `_targets.R`. That would increase visibility so that it's easy for anybody coming after you to add or remove sites. The same would be true for the `parameterCd`, `startDate`, and `endDate`. In general the `_targets.R` is the first place people will look for hardcoded information which might change as a pipeline evolves

Reply:
I decided to add them as default parameter inputs in `download_nwis_data()` for now. Rather than creating additional variable targets for `parameterCD`, `startDate`, and `endDate` we can see the default parameters for the download_mwis_data() function, isn't that easier?